src/madhuraMJun20/BankingSystem.java

```diff
@@ -0,0 +1,56 @@
+package madhuraMJun20;
+
+public class BankingSystem {
+	
+	int counterD=0;
+	int counterC=0;
+	int counterP=0;
+	static int countD=0;
+	static int countC=0;
+	static int countP=0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Done.